misc/labels.md

```diff
@@ -78,22 +72,35 @@ This label is also used for labels that may concern a lot of vis.js users.
 ### `IE / Edge`
 These issues concern a problem with the Microsoft Internet Explorer or Edge browser.<br>
 
+### `invalid`
+This is not a valid issue.<br>
+Someone just created an empty issue, picked the wrong project or something similar.<br>
+This can also be used for pull-request to a non-develop branch or something similar.<br>
+This issue or pull request should be closed immediately.
+
 ### `Issue Inactive`
-Issues marked as `question` or `problem` get marked as inactive when the author is not responsive or the topic is old.<br>
+Issues marked as [`Question`](#question) or [`Problem`](#problem) get marked as inactive when the author is not responsive or the topic is old.<br>
 If an issue is marked as inactive for about 2 weeks it can be closed without any hesitation.
 
 ### `PRIORITY`
-In general this is used for major bugs. There should only exist one or two issues marked as PRIORITY at the same time.<br>
+In general this is used for major bugs. There should only exist a few issues marked as PRIORITY at the same time.<br>
 These issues need to be handled before all others.
 
 ### `Requires breaking change`
 A lot of code needs to be changed to implement this. This is maybe something for a major release or something for someone with a lot of time on their hands :-)
 
-### `invalid`
-This is not a valid issue.<br>
-Someone just created an empty issue, picked the wrong project or something similar.<br>
-This can also be used for pull-request to a non-develop branch oder something similar.<br>
-This issue or pull request should be closed immediately.
-
 ### `waiting for answer/improvement`
 This is mostly used for pull requests were a reviewer requested some changes and the owner has not responded yet.
+
+### `Work In Progress`
+Someone is working on this issue or a pull request already exists and needs to be reviewed.<br>
+
+## Example Workflows
```

Comment:
Love it!

I'd like to see the following or similar here as well (where 'Close' is not a label):

### No response
Feature Request -> Issue Inactive -> Close